tests/compiler/class-implements.release.wat

```diff
@@ -1703,37 +1703,43 @@
        block $class-implements/A3
         block $class-implements/I2
          block $class-implements/A2
-          block $class-implements/F
-           block $class-implements/E
-            block $class-implements/J
-             block $class-implements/D
-              block $class-implements/B
-               block $class-implements/C
-                block $class-implements/I
-                 block $class-implements/A
-                  block $~lib/arraybuffer/ArrayBufferView
-                   block $~lib/string/String
-                    block $~lib/arraybuffer/ArrayBuffer
-                     block $~lib/object/Object
-                      local.get $0
-                      i32.const 8
-                      i32.sub
-                      i32.load $0
-                      br_table $~lib/object/Object $~lib/arraybuffer/ArrayBuffer $~lib/string/String $~lib/arraybuffer/ArrayBufferView $class-implements/A $class-implements/I $class-implements/C $class-implements/B $class-implements/D $class-implements/J $class-implements/E $class-implements/F $class-implements/A2 $class-implements/I2 $class-implements/A3 $class-implements/A4 $class-implements/B2 $class-implements/B3 $class-implements/B4 $invalid
+          block $class-implements/K
```

Comment:
This isn't really caused by this PR (alone), but it would be nice if these cases for interfaces could be removed someday.